api/python/quilt3/main.py

```diff
@@ -129,14 +161,20 @@ def create_parser():
 
     # catalog
     shorthelp = "Run Quilt catalog locally"
-    config_p = subparsers.add_parser("catalog", description=shorthelp, help=shorthelp, allow_abbrev=False)
-    config_p.add_argument(
-        "s3_url",
-        help="S3 URL to browse in local catalog",
-        type=str,
-        nargs="?"
+    catalog_p = subparsers.add_parser("catalog", description=shorthelp, help=shorthelp, allow_abbrev=False)
+    catalog_p.add_argument(
+            "s3_url",
+            help="S3 URL to browse in local catalog",
+            type=str,
+            nargs="?"
+    )
+    catalog_p.add_argument(
+            "--detailed_help",
```

Comment:
Just an idea; not religious. Or `--man-page` idk.
```suggestion
            "--verbose",
```